scripts/release/sign.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -z "${1:-}" ]; then
+  >&2 echo "First argument must be a tag (e.g. v0.2.2)"
+fi
+
+tag="$1"
+system="$(nix eval --raw --impure --expr builtins.currentSystem)"
+prefix="${tag}-${system}"
+release_dir_base="releases/bins"
+release_dir="${release_dir_base}/${prefix}"
+sha256sum_path="releases/${prefix}.SHA256SUMS"
+
+mkdir -p "${release_dir}"
+
+
+>&2 echo "Building..."
+for out in fedimint-pkgs gateway-pkgs ; do
+  nix build "git+file:.?ref=refs/tags/${tag}#${out}"
```

Comment:
I was wondering why we aren't using this build command in the context of #4305, but would these binaries run on non-nix systems? I doubt it and that's why we use bundlers. We should probably rather sign the bundler-generated binaries that we actually publish.

Reply:
Unclear if these binaries are reproducible but let's try. :D

Reply:
https://github.com/fedimint/fedimint/pull/4372